lib/ansible/plugins/callback/log_plays.py

```diff
@@ -9,23 +10,95 @@
     callback: log_plays
     type: notification
     short_description: write playbook output to log file
-    version_added: historical
+    version_added: 2.6
```

Comment:
the new options require the version_added, the plugin itself has been here before 2.6